recipes/nltk-data/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "nltk" %}
+{%set version = "2017.10.22" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "14b55df9314ece2e7e53258c41b2f29a1fe0d7517ded586fa9ed137edf9a39ec" %}
+
+package:
+  name: {{ name }}-data
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/nltk/nltk_data/archive/gh-pages.zip
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+    - unzip
+  run:
+    - python
+    - nltk >=3.2.5
+    - six
+
+test:
+  imports:
+    - nltk
+  files:
+    - test_data.py
+  commands:
+    - python test_data.py
```

Comment:
You could rename `test_data.py` to `run_test.py`, and then drop the whole `test` section entirely; conda automatically runs `run_test.py` files.

Reply:
Nice, didnt knew that. its done.